app/views/frame-connect.jade

```diff
@@ -10,6 +10,8 @@ div(ng-controller="AuthCtrl")
                 h3 Connect to Neo4j
                 p.lead
                   | Database access requires an authenticated connection.
+                p.warn(ng-show="neo4j.clusterRole && failed_connection_attempts > 2")
+                  | Note: User management events do not propagate when running a core-edge cluster. Does your user exist on this server instance?
```

Comment:
I realised that we won't know any information about the server at this point. This message will appear any time a user fails to login more than three times. Switching the sentences around to be 'Does your user exist here? Recall that user management events do not propagate across a cluster' would cover the more likely case that the user doesn't exist at all as opposed to it being a propagation issue.